File: CWEB 2116 - Homework 2/OperationsApplication/PerformTemperatureConversion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OperationsApp
{
	class PerformTemperatureConversion
	{
		public void performTemperatureConversion()
		{
			bool tryagain = true;

			while (tryagain)
			{
				Console.WriteLine("Do you wish to convert: ");
				Console.WriteLine("1. Celsius to Fahrenheit");
				Console.WriteLine("2. Fahrenheit to Celsius");
				Console.WriteLine("3. Exit");
				string answer = Console.ReadLine();

				switch (answer)
				{
					case "1":
						Console.WriteLine("\nEnter Celsius temperature to convert: ");
						double celsius = double.Parse(Console.ReadLine());
						double fahrenheitRes = OperationsLibrary.TemperatureConverter.CelsiusToFahrenheit(celsius);
						Console.WriteLine("Temperature in Celsius = " + celsius);
						Console.WriteLine("Temperature in Fahrenheit = " + fahrenheitRes + "\n");
						break;
					case "2":
						Console.WriteLine("\nEnter Fahrenheit temperature to convert: ");
						double fahrenheit = double.Parse(Console.ReadLine());
						double celsiusRes = OperationsLibrary.TemperatureConverter.FahrenheitToCelsius(fahrenheit);
						Console.WriteLine("Temperature in Fahrenheit = " + fahrenheit);
						Console.WriteLine("Temperature in Celsius = " + celsiusRes + "\n");
						break;
					case "3":
						tryagain = false;
						break;
					default:
						Console.WriteLine("Invalid choice. Try again\n");
						break;
				}
			}
        }
	}
}

File: CWEB2116RD2/src/operationsApplication/PerformTemperatureConversion.java
package operationsApplication;

import java.util.Scanner;

public class PerformTemperatureConversion {
	
	Scanner scanner = new Scanner(System.in);
	
	public void performConversion () {
		
		boolean keepGoing = true;
		
		while (keepGoing) {
			
			System.out.println("Do you wish to convert to ");
			System.out.println("1. Celsius");
			System.out.println("2. Fahrenheit");
			String answer = scanner.nextLine();
			
			switch (answer)
			{
			case "1":
				System.out.println("Enter temperature to convert to Celsius: ");
				double ftemp = scanner.nextDouble();
				double cresult = operationsLibrary.TemperatureConverter.FahrenheitToCelsius(ftemp);
				System.out.println("Fahrenheit Temperature: " + ftemp);
				System.out.println("Celsius Temperature: " + cresult);
				keepGoing = false;
				break;
			case "2":
				System.out.println("Enter temperature to convert to Fahrenheit: ");
				double ctemp = scanner.nextDouble();
				double fresult = operationsLibrary.TemperatureConverter.CelsiusToFahrenheit(ctemp);
				System.out.println("Celsius Temperature: " + ctemp);
				System.out.println("Fahrenheit Temperature: " + fresult);
				keepGoing = false;
				break;
			default:
				System.out.println("Please enter a valid answer\n");
			}
		}
	}
}
